.github/workflows/ci.yml

```
@@ -18,6 +18,9 @@ on:
 jobs:
   validate:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      actions: write  #? For node caching
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)